packages/inventory-vulnerabilities/src/Components/SelectAllCheckbox.js

```diff
@@ -0,0 +1,141 @@
+import { Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { Component } from 'react';
+
+class IndeterminateCheckbox extends React.Component {
+    componentDidMount() {
+        const { indeterminate } = this.props;
+        if (indeterminate) {
+            this._setIndeterminate(true);
+        }
+    }
+
+    componentDidUpdate(previousProps) {
+        if (previousProps.indeterminate !== this.props.indeterminate) {
+            this._setIndeterminate(this.props.indeterminate);
+        }
+    }
+
+    _setIndeterminate(indeterminate) {
```

Comment:
Not really needed anymore, check new version of checkboxes it allows indeterminate state as well.